tasklib/tests.py

```diff
@@ -124,7 +126,8 @@ def test_filtering_by_attribute(self):
 
         # Assert that the correct tasks are returned
         high_priority_task = self.tw.tasks.get(priority="H")
-        self.assertEqual(high_priority_task['description'], "high priority task")
+        self.assertEqual(high_priority_task['description'],
+                         "high priority task")
```

Comment:
Again, it's always good to break after the opening parenthesis in method calls like this, and especially in tests with `assertEqual` I find it much easier to see what's going on (also prefer single quotes as before):

```python
self.assertEqual(
    high_priority_task['description'],
    'high priority task',
)
```

Reply:
Fixed in 8f4fabf